examples/IBFE/explicit/ex5/input2d.iim

```diff
@@ -0,0 +1,248 @@
+GROW_DT                    = 2.0e0             // growth factor for timesteps
+CONVECTIVE_TS_TYPE         = "ADAMS_BASHFORTH" // convective time stepping type
+CONVECTIVE_OP_TYPE         = "STABILIZED_PPM"              // convective differencing discretization type
+CONVECTIVE_FORM            = "ADVECTIVE"       // how to compute the convective terms
+NORMALIZE_PRESSURE         = FALSE             // whether to explicitly force the pressure to have mean zero
+ERROR_ON_DT_CHANGE         = TRUE              // whether to emit an error message if the time step size changes
+VORTICITY_TAGGING          = TRUE              // whether to tag cells for refinement based on vorticity thresholds
+TAG_BUFFER                 = 4                 // size of tag buffer used by grid generation algorithm
+REGRID_CFL_INTERVAL        = 0.5               // regrid whenever any material point could have moved 0.5 meshwidths since previous regrid
+OUTPUT_U                   = TRUE
+OUTPUT_P                   = TRUE
+OUTPUT_F                   = TRUE
+OUTPUT_OMEGA               = TRUE
+OUTPUT_DIV_U               = TRUE
+ENABLE_LOGGING             = TRUE
+
+// model parameters
+KAPPA_S_BODY = 0.0 //2.5*DX/DT^2
+ETA_S_BODY = 0.0 //1.5*DX/DT
+KAPPA_S_SURFACE =0.0001/DT^2 //100 //0.05 *DX/DT^2
+ETA_S_SURFACE = 0.0004/DT
+C1_S = 0.0
+
+
+
+// collocated solver parameters
+PROJECTION_METHOD_TYPE = "PRESSURE_UPDATE"
+SECOND_ORDER_PRESSURE_UPDATE = TRUE
+
+VelocityBcCoefs_0 {
+   t_half = 0.5
+   tau = 0.25
+
+   acoef_function_0 = "1.0"
+   acoef_function_1 = "0.0"
+   acoef_function_2 = "0.0"
+   acoef_function_3 = "0.0"
+
+   bcoef_function_0 = "0.0"
+   bcoef_function_1 = "1.0"
+   bcoef_function_2 = "1.0"
+   bcoef_function_3 = "1.0"
+
+   gcoef_function_0 = "(tanh(t/2-t_half/tau)+tanh(t_half/tau))/(1+tanh(t_half/tau))"
+   gcoef_function_1 = "0.0"
+   gcoef_function_2 = "0.0"
+   gcoef_function_3 = "0.0"
+}
+
+VelocityBcCoefs_1 {
+
+	L = L
+   acoef_function_0 = "1.0"
+   acoef_function_1 = "0.0"
+   acoef_function_2 = "1.0"
+   acoef_function_3 = "1.0"
+
+   bcoef_function_0 = "0.0"
+   bcoef_function_1 = "1.0"
+   bcoef_function_2 = "0.0"
+   bcoef_function_3 = "0.0"
+
+   gcoef_function_0 = "cos(pi*X_1/L)*exp(-2.0*t)"
+   gcoef_function_1 = "0.0"
+   gcoef_function_2 = "0.0"
+   gcoef_function_3 = "0.0"
+}
+
+IBHierarchyIntegrator {
+   start_time          = START_TIME
+   end_time            = END_TIME
+   grow_dt             = GROW_DT
+   regrid_cfl_interval = REGRID_CFL_INTERVAL
+   dt_max              = DT
+   error_on_dt_change  = ERROR_ON_DT_CHANGE
+   enable_logging      = ENABLE_LOGGING
+}
+
+IBFESurfaceMethod {
+   IB_delta_fcn                = IB_DELTA_FUNCTION
+   split_forces                = SPLIT_FORCES
+   use_pressure_jump_conditions   = USE_PRESSURE_JUMP_CONDITIONS
+   use_velocity_jump_conditions   = USE_VELOCITY_JUMP_CONDITIONS
+   use_l2_lagrange_family = USE_L2_LAGRANGE_FAMILY
+   use_pointwise_jump_imposition = USE_POINTWISE_JUMP_IMPOSITION
```

Comment:
I should have removed the flag `use_pointwise_jump_imposition = USE_POINTWISE_JUMP_IMPOSITION` since the point wise jump conditions have been removed from the code.

Reply:
OK --- please make a commit that updates to the input files and source code as needed. I will squash together the commits once everything is finished.

Reply:
Sounds good..Im working on it  now.